### PR TITLE
Finish List & Query api

### PR DESCRIPTION
**Pull request by zzihan1203**
Need to talk about the error handler later.

### CODE REVIEW

File: tests/test_service.py [hunk 1]
```diff
@@ -94,9 +94,12 @@ def test_query_cart_iterms(self):
         test_customer_id = shopcarts[0].customer_id
         test_target_price = shopcarts[0].price
         customer_id_shopcarts = [shopcart for shopcart in shopcarts if shopcart.customer_id == test_customer_id and shopcart.price <= test_target_price]
+        print(customer_id_shopcarts)
```
Comment (ytc338) [inline tests/test_service.py]: remove these print statements

File: service/service.py [hunk 2]
```diff
@@ -93,55 +93,71 @@ def index():
...
-    else:    
-        target_price = request.args.get('price')
-        app.logger.info('Request to query all items in shopcart with customer_id: %s', customer_id)
-        items = []
-        items = Shopcart.query_by_target_price(customer_id, target_price)
-        results = [item.serialize() for item in items]
-        return make_response(jsonify(results), status.HTTP_200_OK)
+@api.route('/shopcarts/<int:customer_id>', strict_slashes=False)
+@api.param('customer_id', 'Customer Identifier')
+class shopcartsCollection(Resource):
```
Comment (ytc338) [inline service/service.py]: I think we have no concept of collection so can we name it ShopcartItem? And the naming convention is to start with upper case

**Review by ytc338 — APPROVED** [review]

**Comment by rishabhydv** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/05d659ad-a782-439b-834f-3389ea91917f?env_id=ibm%3Ayp%3Aus-south "nyu-shopcart-service-f19-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/a50ef012-23d6-4da0-a27b-b570a2cc224c "nyu-shopcart-service-f19-pipeline") deployed [shopcarts](https://github.com/NYUDevops-ShopCart/shopcarts/releases/tag/deploy-ibm-yp-us-south-rr3591-nyu.edu-dev-20191211-003507 "deploy-ibm-yp-us-south-rr3591-nyu.edu-dev-20191211-003507") to [dev](https://console.bluemix.net/apps/8a88bfb7-b467-43b2-b23e-9a30dd80d2e2?env_id=ibm:yp:us-south "ibm:yp:us-south:rr3591@nyu.edu:dev"), including commit 68be12c98ef94bb0395f3b366bcbfda7efc23df1

**Comment by rishabhydv** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/05d659ad-a782-439b-834f-3389ea91917f?env_id=ibm%3Ayp%3Aus-south "nyu-shopcart-service-f19-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/a50ef012-23d6-4da0-a27b-b570a2cc224c "nyu-shopcart-service-f19-pipeline") deployed [shopcarts](https://github.com/NYUDevops-ShopCart/shopcarts/releases/tag/deploy-ibm-yp-us-south-rr3591-nyu.edu-prod-20191211-005015 "deploy-ibm-yp-us-south-rr3591-nyu.edu-prod-20191211-005015") to [prod](https://console.bluemix.net/apps/34d9c22a-fc58-43ca-9e90-6702e70d3e0f?env_id=ibm:yp:us-south "ibm:yp:us-south:rr3591@nyu.edu:prod"), including commit 68be12c98ef94bb0395f3b366bcbfda7efc23df1

**Comment by rishabhydv** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/05d659ad-a782-439b-834f-3389ea91917f?env_id=ibm%3Ayp%3Aus-south "nyu-shopcart-service-f19-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/a50ef012-23d6-4da0-a27b-b570a2cc224c "nyu-shopcart-service-f19-pipeline") deployed [shopcarts](https://github.com/NYUDevops-ShopCart/shopcarts/releases/tag/deploy-ibm-yp-us-south-rr3591-nyu.edu-dev-20191211-012419 "deploy-ibm-yp-us-south-rr3591-nyu.edu-dev-20191211-012419") to [dev](https://console.bluemix.net/apps/8a88bfb7-b467-43b2-b23e-9a30dd80d2e2?env_id=ibm:yp:us-south "ibm:yp:us-south:rr3591@nyu.edu:dev"), including commit 68be12c98ef94bb0395f3b366bcbfda7efc23df1

**Pull request merged**